inject/src/main/java/io/avaje/inject/BeanScope.java

```diff
@@ -33,33 +33,24 @@
  *
  * }</pre>
  *
- * <h3>External dependencies</h3>
+ * <h3>Runtime dependencies</h3>
```

Comment:
I don't think this is "Runtime" ... it is "Externally provided" to the Bean Scope